code/modules/reagents/chemistry/recipes/drinks.dm

```diff
@@ -893,3 +893,11 @@
 	result_amount = 3
 	mix_message = "The ingredients mix into a dark brown godly substance"
 	mix_sound = 'sound/goonstation/misc/drinkfizz.ogg'
+
+/datum/chemical_reaction/dark_and_stormy
+	name = "Dark and stormy"
```

Comment:
Probably doesn't matter with the chemical reaction name, but may as well do this too for consistency.
```suggestion
	name = "Dark and Stormy"
```